docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Acquisition experiences - .NET Core
+description: .NET Core Acquisition Experiences
+author: jamshedd
+ms.author: jamshedd
+ms.date: 09/06/2019
+---
+# .NET Core acquisition experiences
+
+There are various Acquisition options for .NET Core - 
+* [Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)
+* [https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)
+* [Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)
+* [Docker](#Docker)
+* [Linux Package Managers](#Linux-Package-Managers)
+* [Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)
+* [Snap store](#Snap-store)
+* [.NET Core on GitHub](#NET-Core-on-GitHub)
+
+Each of these choices has their own strengths. Some options support both initial acquisition and subsequent notification of updates for the .NET Core runtime and tools, while others don't support update notifications.
+
+|Acquisition Choice|Update|
+|:-:|:-:|
+[Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)   | ✔️ |
+[https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)        | ❌ |
+[Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)                | ✔️ |
+[Docker](#Docker)                                                               | ✔️ |
+[Linux Package Managers](#Linux-Package-Managers)                               | ✔️\*|
+[Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)   | ✔️ |
```

Comment:
This ties back to your previous comment (replacing updates with update notifications). When I built the table I was thinking of updates to Core, not update notifications specifically. When thinking about updates the installer task does allow us to get the latest update without having to call out the specific update (version=2.x) which would give the latest minor for that major.  I am inclined to revert my first change to go back to Updates since that encompasses more than update notifications.